stable/airflow/README.md

```diff
@@ -291,6 +291,9 @@ The following table lists the configurable parameters of the Airflow chart and t
 | `airflow.extraContainers`                | additional containers to run in the scheduler, worker & web pods | `[]`             |
 | `airflow.extraVolumeMounts`              | additional volumeMounts to the main container in scheduler, worker & web pods | `[]`|
 | `airflow.extraVolumes`                   | additional volumes for the scheduler, worker & web pods | `[]`                      |
+| `flower.resources`                       | enable workers                                          | `{}`                      |
```

Comment:
Can you update the doc and add add resource for worker as well

Reply:
Updated the documentation, the workers already had resources added in the config :)